plugins/web-plugin/src/main/java/ru/sbtqa/tag/pagefactory/web/drivers/WebDriverService.java

```diff
@@ -90,6 +92,9 @@ private void createDriver() throws UnsupportedBrowserException, MalformedURLExce
             } else if (browserName.equals(BrowserName.INTERNET_EXPLORER)) {
                 WebDriverManagerConfigurator.configureDriver(InternetExplorerDriverManager.getInstance(), BrowserName.IE.getName());
                 setWebDriver(new InternetExplorerDriver(capabilities));
+            } else if (browserName.equals(BrowserName.EDGE)) {
+                WebDriverManagerConfigurator.configureDriver(EdgeDriverManager.getInstance(), BrowserName.EDGE.getName());
+                setWebDriver(new EdgeDriver(capabilities));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "EdgeDriver"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=149&issues=AWhKiWL7rt4Kct_4ro40&open=AWhKiWL7rt4Kct_4ro40)

[ISSUE_KEY]: AWhKiWL7rt4Kct_4ro40